sort/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace sort
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            int[] x = new int[] { 14, 4, 17, 5, 8, 2, 1, 3, 9 };
+            int temp = 0;
```

Comment:
Great job! The only improvement I can see you could make here, would be to set this to `int.MinValue` instead of `0`. That way you always start with the lowest and you can work with negative numbers!